go.graphics.swing/src/go/graphics/swing/AreaContainer.java

```diff
@@ -35,6 +28,13 @@
 import com.jogamp.opengl.awt.GLJPanel;
 import com.jogamp.opengl.glu.GLU;
 
+import go.graphics.RedrawListener;
+import go.graphics.area.Area;
+import go.graphics.event.GOEvent;
+import go.graphics.event.GOEventHandlerProvider;
+import go.graphics.swing.event.swingInterpreter.GOSwingEventConverter;
+import go.graphics.swing.opengl.JOGLDrawContext;
+
```

Comment:
@andreasb242: It isn't dramatic, but in the future, please recheck your commits and leave out changes like this, which aren't really a change. They only make the diff larger and more confusing.